### DISCUSSION TITLE
Signup

### DISCUSSION THREAD

**Discussion by fokewulf** [thedevs-network/kutt#594] [General]
Googmorning,
when re enable signup??

Thanks, Daniele

**Comment by poeti8** [comment]
Until I have a paid 24/7 back-office team where they can monitor and ban malware/scam links which itself require me to generate money from this project so I can be able to pay them, so yeah, not anytime soon.

**Reply by qplus7** [reply]
I just installed this and ran into issues signing up. I thought I would be signing up on my own server, and maybe I am, but it doesn't seem like it based on what I'm seeing here. Is this project dead?
